t/rose-metadata-check/lib/custom_widget.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, value, metadata, set_value, hook, arg_str=None):
             tooltip_text = self.WARNING_MESSAGE.format(value,
                                                        int_value)
         my_adj = gtk.Adjustment(value=int_value,
-                                upper=sys.maxint,
-                                lower=-sys.maxint - 1,
+                                upper=sys.maxsize,
+                                lower=-sys.maxsize - 1,
```

Comment:
Not quite the same. `sys.maxsize` returns the largest container size.

Reply:
Removed this file...